src/ur_arm.cpp

```diff
@@ -153,16 +153,35 @@ struct UR5eArm::state_ {
     std::string output_csv_dir_path;
 };
 
-enum class UR5eArm::UrDriverStatus : int8_t  // Only available on 3.10/5.4
+enum class URArm::UrDriverStatus : int8_t  // Only available on 3.10/5.4
 {
     NORMAL = 1,
     ESTOPPED = 2,
     READ_FAILURE = 3,
     DASHBOARD_FAILURE = 4
 };
 
-UR5eArm::UR5eArm(const Dependencies& deps, const ResourceConfig& cfg) : Arm(cfg.name()) {
-    VIAM_SDK_LOG(info) << "UR5eArm constructor start";
+std::vector<std::shared_ptr<ModelRegistration>> URArm::create_model_registrations() {
+    using namespace std::placeholders;
+
+    constexpr auto arm_factory = [](Model model, const Dependencies& deps, const ResourceConfig& config) {
+        return std::make_unique<URArm>(std::move(model), deps, config);
+    };
+
+    const auto arm = API::get<Arm>();
+    const auto registration_factory = [&](const Model& model) {
+        return std::make_shared<ModelRegistration>(arm, model, std::bind(arm_factory, model, _1, _2));
+    };
+
+    const auto family = ModelFamily{"viam", "universal-robots"};
+    return {
+        registration_factory({family, "ur5e"}),
+        registration_factory({family, "ur20"}),
```

Comment:
we should go ahead and add this to the meta.json. we can prob wait to update the readme before we release

Reply:
Done